### ISSUE TITLE
Plots

### ISSUE THREAD

**Issue by rufo123** [ForgeEssentials/ForgeEssentials#1991]
Plots are created randomly on server. Mostly in someone other plot. [In latest build]

**Comment by DPxKnightxZero** [comment]
I can confirm this bug. Suddenly my 1000 x 1000 plot is now owned by like 10 people. Its sad cause I spent a lot of time gather "Ultinium's" server currency to get it. Each kill is worth 10 "Ultinium's"

**Comment by olee** [comment]
Wait - you mean plots are OWNED by multiple people?
This should not be possible... I'm pretty sure about that.

**Comment by DPxKnightxZero** [comment]
What information do you need? I will get it within the hour.

**Comment by DPxKnightxZero** [comment]
It is an issue! Please fix plots randomly changing plots! "Currently testing with http://ci.forgeessentials.com/job/FE/1154/" hopefully this is fixed.

**Comment by MickScandalous** [comment]
I can confirm this as well, and When I leave a plot it tells me im entering another plot that is in another location on the world, its on the same Y-axis though if that makes a difference

**Comment by Zethalion** [comment]
As we moved to 1.12.2, is this still an issue? Closing this issue in 7 days if no response @rufo123

**Comment by MickScandalous** [comment]
Yes it is still an on going issue as far as I can tell, let me know if you fix it in the next  2 1/2 years lol.
